src/main/java/core/basesyntax/LocalFileReader.java

```diff
@@ -0,0 +1,63 @@
+package core.basesyntax;
+
+import java.io.BufferedReader;
+import java.io.FileReader;
+import java.io.IOException;
+import java.time.LocalDate;
+import java.time.format.DateTimeParseException;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.IllegalFormatFlagsException;
+import java.util.List;
+
+public class LocalFileReader {
+    private static final String COMMA_DELIMITER = ",";
+    private static final String[] FILE_HEADER = new String[]{"type", "fruit", "quantity", "date"};
+    private final String filePath;
+
+    public LocalFileReader(String filePath) {
+        this.filePath = filePath;
+    }
+
+    public List<List<String>> readFromFile() throws IOException {
```

Comment:
```suggestion
    public List<Record/Transaction> readFromFile() throws IOException {
```
ideally we would parse each row into record model with fields: quantity, date, name, etc. that will make code easier to use. But it's up to you in this case

Reply:
отписал в личку: хочу на выходных посидеть, поскольку интресено несколько иначе реализовать логику обработки и хранения операций (покупка, продажа, возврат). а суетиться, чтоб успеть к дэдлайну, не охота. если это не принципиально, пока предпочел бы оставить так

Reply:
done